addons/binding/org.openhab.binding.nest.test/OSGI-INF/org.openhab.binding.nest.test.NestTestHandlerFactory.xml

```diff
@@ -1,4 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+	Copyright (c) 2010-2018 by the respective copyright holders.
+
+	All rights reserved. This program and the accompanying materials
+	are made available under the terms of the Eclipse Public License v1.0
+	which accompanies this distribution, and is available at
+	http://www.eclipse.org/legal/epl-v10.html
+
+-->
 <scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.2.0" configuration-pid="binding.nest.test" configuration-policy="optional" immediate="true" name="org.openhab.binding.nest.test.NestTestHandlerFactory">
```

Comment:
NestTestHandlerFactory has DS annotations but this file is committed because of https://github.com/openhab/openhab-core/issues/318. So if you add the header Eclipse will regenerate it without the header and it ends up in the changed files without people having really modified it. To fix that maybe you can also remove the DS annotations from NestTestHandlerFactory?

Reply:
Just for my understanding: Why remove the DS annotations and not remove the xml instead?

Reply:
Because then the Maven build would fail because the maven-scr-plugin plugin does not generate XML for test bundles (https://github.com/openhab/openhab-core/issues/318).

Reply:
Ah, thanks!